internal/api/ui/login/static/templates/change_password.html

```diff
@@ -12,7 +12,7 @@ <h1>{{t "PasswordChange.Title"}}</h1>
     {{ .CSRF }}
 
     <input type="hidden" name="authRequestID" value="{{ .AuthReqID }}" />
-    <input type="hidden" name="loginname" value="{{ .LoginName }}" autocomplete="username" />
+    <input type="text" name="loginname" value="{{ .LoginName }}" autocomplete="username" class="hidden" />
```

Comment:
Is this still used? It seems the login UI does not provide me with the option to change password. Only inside the console.